migrations/versions/a38a346e6ded_create_refresh_token_table.py

```diff
@@ -0,0 +1,42 @@
+"""Create refresh_token table
+
+Revision ID: a38a346e6ded
+Revises: 
+Create Date: 2020-03-15 19:26:18.544300
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "a38a346e6ded"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # check whether the `refresh_token` table already exists. We should
+    # not try to re-create the table in deployments whose DB was created
+    # before this "initial state" migration code was added.
+    conn = op.get_bind()
+    inspector = sa.engine.reflection.Inspector.from_engine(conn)
+    tables = inspector.get_table_names()
+    if "refresh_token" in tables:
+        return
+
+    op.create_table(
```

Comment:
Instead of manually doing so, I think we could just define a SQLAlchemy core `Table` here and use its `create()` function, which will do the check for you.

Reply:
thanks for the suggestions - noted, but since this works and it's blocking the QA team i will merge it as it is